### PR TITLE
petritrailerlines

### PR DESCRIPTION
**Pull request by Petrikow**
Just a few changes for the trailer. Put your best foot forward, etc

### CODE REVIEW

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1G.txt [hunk 1]
```diff
@@ -946,7 +946,7 @@ Her body rises, attempting to push back against the wall of light.
 [sha:ff1023ded32bc7ca0051bf31b7e045f8329d6bc8]{
 -- Page 331, Offset 39321.
 -- 「ええ。人間では神の如きモノは倒せない。
-"Indeed. Man cannot fell those akin to a god.
+"Indeed. Man cannot fell those in the realm of a god.
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01B_01_CIEL14B_1G.txt:949]: Original sounds better to me here

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 2]
```diff
@@ -762,7 +762,7 @@ Having obtained the body that had once been mine,
 [sha:d62390bde028dbf1f7d9b7efc6f972c17dd5f913]{
 -- Page 542, Offset 19659.
 -- ……そして、あってはならない事変は収束した。
-And then... the incident that should never have been came to its conclusion.
+And then, the reprehensible incident came to its conclusion.
```
Comment (rschlaikjer) [inline script/Ciel/Day 8/08_01_CIEL08_6.txt:765]: Reprehensible sounds very headmastery / weak to me. Prefer original or something meatier.

Comment (Petrikow) [reply script/Ciel/Day 8/08_01_CIEL08_6.txt:765]: I don't like these あってはならない constructions as 'must not', 'can't', but on the other hands they're tricky in general. Very Japanese. I guess "unspeakable" maybe works?

Comment (rschlaikjer) [reply script/Ciel/Day 8/08_01_CIEL08_6.txt:765]: Yeah, I think that works for me. Reprehensible just makes it sound like the incident has been a very naughty boy.

**Pull request merged**